packages/utils/perf/score-report/optimized1.ts

```diff
@@ -1,5 +1,12 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
The rules can be on one line. This is the syntax, I think:

```suggestion
/* eslint-disable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-assignment */
```